### DISCUSSION TITLE
Alpine Vaultwarden Errors

### DISCUSSION THREAD

**Discussion by nicedevil007** [tteck/Proxmox#1132] [General]
![image](https://user-images.githubusercontent.com/17103076/223511615-37a3b959-19e4-4d0e-a876-62c2a9486559.png)

Not updateable right now, or the script isn't installing the latest version.

Also it is not possible to use Organizations now. I can create one but I can't invite someone else or even see my imported credentials from the previous bitwarden installation

stuck on this loading animation:

![image](https://user-images.githubusercontent.com/17103076/223512308-ef87beaa-92b7-449c-9519-77cd62ef2342.png)

**Comment by tteck** [comment]
>Not updateable right now, or the script isn't installing the latest version.

The server checks against the latest commit on GitHub, so it will show an 'Update' even if it's already on the latest version. The build version is pulled directly from GitHub instead of using a pre-packaged version.

**Comment by tteck** [comment]
Keep an eye on the storage usage. By default, the storage is set to 300MiB unless you manually change it using advanced settings. The other install method is using 6GB.

**Reply by nicedevil007** [reply]
Plenty of room there
![image](https://user-images.githubusercontent.com/17103076/223526487-faf6a0d2-ce2f-41fe-a75e-95e7fc5f226f.png)

so I would assume that this is not the problem for the issue with organizations

**Reply by nicedevil007** [reply]
ok tryed it with 2 gb for the bootdrive now, no change, tryed it with 512 mb memory, no change

**Reply by nicedevil007** [reply]
nobody else got this error?

**Reply by tteck** [reply]
On the Organization page, I see the spinning wheel, but all the imported items are visible in the vault. I don't send invitations. The package for Vaultwarden on https://repology.org/project/vaultwarden/packages is pre-packaged, so I have no control.

**Reply by nicedevil007** [reply]
So I Open an issue on vaultwarden GitHub. Organizations are there to share credentials easy :/ btw where did the /opt/vaultwarden folder move?

**Comment by nicedevil007** [comment]
So the guys from vaultwarden are telling me that we are not using the mainbranch in alpine Linux, that's why we are facing this error. What now?

https://github.com/dani-garcia/vaultwarden/issues/3330

**Reply by tteck** [reply]
I set the script to download and use web-vault from https://github.com/dani-garcia/bw_web_builds  https://github.com/tteck/Proxmox/commit/119b1c07911db0eab7ca01709a4d0467c6f377c0
Still have the spinning wheel 🤷🏻‍♂️

**Reply by nicedevil007** [reply]
I'm about to post this issue on gitlab from alpine linux.
I went through a complete manual installation of a LXC with alpine and then installed it with the testing repo. So maybe a guy from there can find the fix.

This issue wasn't there with the same version of vaultwarden (1.27, webvault 2023.2) on your Debian script, so I assume it isn't an issue from vaultwarden.

**Reply by nicedevil007** [reply]
here we go https://gitlab.alpinelinux.org/alpine/aports/-/issues/14705

**Reply by nicedevil007** [reply]
> I set the script to download and use web-vault from https://github.com/dani-garcia/bw_web_builds [119b1c0](https://github.com/tteck/Proxmox/commit/119b1c07911db0eab7ca01709a4d0467c6f377c0) Still have the spinning wheel 🤷🏻‍♂️

mhhh I tested your fresh update on the script now but it still says webvault version 2023.2 => it should be on 2022.12 right?
![image](https://user-images.githubusercontent.com/17103076/224487435-e91a3d70-1236-4ea3-8699-1434d140e8bf.png)

![image](https://user-images.githubusercontent.com/17103076/224487480-6ee08ca2-9999-4006-9257-fa129b0e420f.png)

latest version with there github release from december 2022
![image](https://user-images.githubusercontent.com/17103076/224487536-d7ce1122-fd5a-4884-ab88-e97a211716be.png)

**Reply by tteck** [reply]
Go to the [web-vault link](https://github.com/dani-garcia/bw_web_builds) to get the latest patched version.

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/1132#discussioncomment-5275568

**Reply by tteck** [reply]
I dislike that it has to use web-vault 2022.12, that's 3 versions behind.

**Reply by nicedevil007** [reply]
Same here, but I know that the devs on vaultwarden may have to adjust there stuff to be able to use all the new versions coming out from Bitwarden themself

With all the Feature we can use without any cost I'm still absolutely fine with that situation :)

**Comment by tteck** [comment]
If organizations are not needed and given there is a current problem with them, I believe that running Vaultwarden without using organizations is a viable approach until a solution is found.

**Reply by nicedevil007** [reply]
no option for me, then I have to move back to the debian version :/ but anyway we will see what the guys on gitlab will tell us

**Comment by nicedevil007** [comment]
Sooo now it got fixed but we have to use the edge kernel at all stages right now, maybe we can change it later on if a stable release will be put out. I learned now what else commands are relevant to get the right repository stuff if we change it, I will make a pull request for your vaultwarden-alpine again.

**Reply by tteck** [reply]
I'll change it

**Reply by towerhand** [reply]
> I'll change it

Do I need to create a new container for the new changes? or just update it with the script inside the container?

**Reply by tteck** [reply]
New creation

**Reply by nicedevil007** [reply]
> I'll change it

sry was too late you already changed it...
wanted to test if it is enough to just change the repostuff for edge/testing and not for community and main

it works with just the edge/testing and without having to rely on edge for main and community stuff. So we make it a bit less unstable :)

**Reply by nicedevil007** [reply]
@towerhand 

you can change the repository stuff as well and dont have to recreate the container:

just run the following lines in your container one time:

```pyhton
cat <<EOF >/etc/apk/repositories
https://dl-cdn.alpinelinux.org/alpine/edge/main
https://dl-cdn.alpinelinux.org/alpine/edge/community
https://dl-cdn.alpinelinux.org/alpine/edge/testing
EOF
apk update
apk upgrade -a
```

Then open a browser without cached old stuff (inprivate mode) to verify it is working.
